syft/core/hooks/torch/guard.py

```diff
@@ -61,11 +59,29 @@ def types_guard(self, torch_type_str):
                 "Tried to receive a non-Torch object of type {}.".format(
                     torch_type_str))
 
-    def tensor_contents_guard(self, contents):
+
+    def tensor_contents_guard(self,contents):
         """tensor_contents_guard(contents) -> contents
-        TODO: check to make sure the incoming list isn't dangerous to use for
+        check to make sure the incoming list isn't dangerous to use for
                constructing a tensor (likely non-trivial). Accepts the list of JSON objects
-               and returns the list of JSON ojects. Should throw and exception if there's a
+               and returns the list of JSON ojects.Throws an exception if there's a
                security concern.
         """
+        
+        is_instance = lambda tensor : all(isinstance(digit, (int, float)) for digit in tensor)
```

Comment:
Can we add `bool` here?  It's possible to construct a binary tensor of integers of various bitwidth, e.g. `torch.ByteTensor([[True, False],[False, True]])` should have the same behavior as `torch.ByteTensor([[1, 0],[0, 1]])`, and similar for IntTensor/LongTensor.